.github/workflows/ci-dgraph-load-tests.yml

```diff
@@ -16,7 +16,7 @@ on:
 jobs:
   dgraph-load-tests:
     if: github.event.pull_request.draft == false
-    runs-on: self-hosted
+    runs-on: [self-hosted, X64]
```

Comment:
small check is needed here - is this an `and` or `or`?

Reply:
AND, confirmed [here.]( https://docs.github.com/en/actions/hosting-your-own-runners/using-self-hosted-runners-in-a-workflow#using-default-labels-to-route-jobs).  However I need to change label to ARM64 which is the default label github actions uses.

Reply:
+1 ^ thanks!